### ISSUE TITLE
DTLS Support ?

### ISSUE THREAD

**Issue by Sirbu** [summitt/Nope-Proxy#73]
Hi, 

I discovered your plugin while searching for a way to proxify UDP traffic. Your plugin support this, and TLS, but not both at the same time.

Will there be a point in the future where this might be the case ?

Best regards

**Comment by summitt** [comment]
Yes, it's on my list of things to add. Just have not found the time to implement it. :( Should be added before the end of the year.

**Comment by delasy** [comment]
@summitt hey Josh, is there a way perhaps to export requests and decrypt them manually? \
do you still plan to implement support for DTLS?
